recipe/meta.yaml

```diff
@@ -1,88 +1,100 @@
-{% set version = "4.2.2" %}
+{% set name = "ffmpeg" %}
+{% set version = "6.1.1" %}
```

Comment:
ffmeg may require a very big rebuild.  Have you identified all the downstream packages that may be effected? This is the reason why it hasn't been updated to the latest is that it will require work and also 4.2.x is actively supported. See -> https://endoflife.date/ffmpeg.

Reply:
Mostly opencv actually, which @skupr-anaconda is working on updating.
https://metayaml-conda.fly.dev/metayaml/reverse_dependencies?name=ffmpeg

Reply:
can I keep working on this then? @skupr-anaconda? Is it safe to update ATM?

Reply:
Yes, ffmpeg has an upper bound >=4.2.2,<4.3.0a0 so it's safe